department-of-veterans-affairs/vets-website
# Added cie and pcie pagerduty services.

## Description
This PR adds the pager duty component to the wrapper component. Also in this PR are some updates to move the footer component into the wrapper, so that the footer can still be displayed when a downtime window is active. 

Lots of unit tests were refactored as a result of adding and moving things at the wrapper level.

**To test:**

- Set a maintenance window for either Staging: Day of Check-in or Staging: Pre-Check-In here: https://dsva.pagerduty.com/service-directory?direction=asc&query=&sort_by=name&team_ids=PPW0SBU
- Add an appointment via the scheduling tool
- Visit the appointment in the review environment deployed by this PR.
- Verify that the downtime message is displayed (*Note it can take up to 6 minutes to show)

## Original issue(s)
department-of-veterans-affairs/va.gov-team#46556
department-of-veterans-affairs/va.gov-team#46559

## Testing done
Added unit test to trigger downtime and updated many other tests.

## Screenshots
![0bf87dd3826d11fc22de6465fc51e8ca review vetsgov-internal_health-care_appointment-pre-check-in_error_error=bad-token](https://user-images.githubusercontent.com/13967174/191072280-8bc2b6a1-2203-4c17-95d6-51730dc6d250.png)


## Definition of done
